photon-client/src/components/dashboard/tabs/TargetsTab.vue

```diff
@@ -2,8 +2,38 @@
 import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
 import { PipelineType } from "@/types/PipelineTypes";
 import { useStateStore } from "@/stores/StateStore";
+import Vue, { ref, watch } from "vue";
+import type { Transform3d } from "@/types/PhotonTrackingTypes";
 
-const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings;
+let oldResults: {targets: Transform3d[]} = Vue.observable({ targets: [] });
+let stdev = ref({
+  x: 0,
+  y: 0,
+  z_angle: 0
+});
+
+const standardDeviation = (arr, usePopulation = false) => {
```

Comment:
type arr as `number[]`

nit: get rid of the `usePopulation `prop as it is always false